ddprof-lib/src/main/java/com/datadoghq/profiler/JavaProfiler.java

```diff
@@ -446,7 +445,9 @@ public Map<String, Long> getDebugCounters() {
     private static native boolean init0();
     private native void stop0() throws IllegalStateException;
     private native String execute0(String command) throws IllegalArgumentException, IllegalStateException, IOException;
-    private native void filterThread0(boolean enable);
+
+    private native void filterThread_add();
+    private native void filterThread_remove();
```

Comment:
Please, use the naming convention
```suggestion
    private native void filterThreadAdd0();
    private native void filterThreadRemove0();
```